web-api/migration-terraform/main/lambdas/migrations/0013-trial-session-default-proceedingType.js

```diff
@@ -0,0 +1,47 @@
+const createApplicationContext = require('../../../../src/applicationContext');
+const {
+  DEFAULT_PROCEEDING_TYPE,
+} = require('../../../../../shared/src/business/entities/EntityConstants');
+const {
+  TrialSession,
+} = require('../../../../../shared/src/business/entities/trialSessions/TrialSession');
+const applicationContext = createApplicationContext({});
+
+const migrateItems = async items => {
+  const itemsAfter = [];
+  for (const item of items) {
+    if (
+      item.pk.startsWith('trial-session|') &&
+      item.sk.startsWith('trial-session|')
+    ) {
+      if (!item.proceedingType) {
+        console.log('adding a proceeding type');
+        const trialSessionEntity = new TrialSession(
+          {
+            ...item,
+            proceedingType: DEFAULT_PROCEEDING_TYPE,
+          },
+          {
+            applicationContext,
+          },
+        );
+
+        itemsAfter.push({
+          ...item,
+          ...trialSessionEntity.validate().toRawObject(),
+        });
+        console.log('pushed an updated item', {
```

Comment:
this feels a little unfinished. there should be just one log entry per operation that clearly describes what action we are taking, and we should use `applicationContext.logger.info(...)` 

In this case, how does this sound?

```
applicationContext.logger.info(
  `migrating trial session ${item.trialSessionId} with default proceeding type: ${DEFAULT_PROCEEDING_TYPE}`, 
  { 
    ...item, 
    ...trialSessionEntity.validate().toRawObject()
  }
);
```

... and we'd lose the `console.log(...)` on line 18